make/source.mk

```diff
@@ -288,6 +288,7 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             common/filter.c \
             common/maths.c \
             common/sdft.c \
+			common/stopwatch.c \
```

Comment:
(wrong whitespace)

Reply:
Good find... I don't know why for makefiles vscode inserts tabs even though I set it to 4 spaces explicitly.